libraries/platform/iot_clock.h

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
I don't think this file should go here. I think we should not have any specific portable abstraction layers. I also do not forsee any future portable layers being added here in this folder. I would vote for this file to be bundled with the logging stuff under:
libraries/utilities/include/iot_clock.h

Reply:
I think that is a good call if the new development will not carry any platform dependencies. Will move the file